### PR TITLE
[cifuzz] Save 1.1GB in image size

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Default gcloud install is bloated with anthos junk. Install gsutil via pip.
This cuts pull time in half - or about 30 seconds.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
I need to test that uploading still works. Downloading does.

**Review by Alan32Liu — APPROVED** [review]

File: infra/build_fuzzers.Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Docker image to run fuzzers for CIFuzz (the run_fuzzers action on GitHub
 # actions).
 
-FROM gcr.io/oss-fuzz-base/cifuzz-base
```
Comment (oliverchang) [inline infra/build_fuzzers.Dockerfile:19]: Why is this change needed?

**Review by oliverchang — COMMENTED** [review]
LGTM with a question

**Comment by oliverchang** [comment]
Do we still need this?

**Comment by jonathanmetzman** [comment]
> Do we still need this?

Yes

File: infra/build_fuzzers.Dockerfile (hunk 1 above)
Comment (jonathanmetzman) [reply infra/build_fuzzers.Dockerfile:19]: So I can test this code.
